fr.json

```diff
@@ -0,0 +1,592 @@
+    "premiumQueueLimitReached": "Vous avez atteint la limite de la file d'attente pour les utilisateurs gratuits. Vous pouvez ajouter jusqu'à 1500 chansons à la file d'attente. Vous pouvez acheter Betty premium sur https://betty.cx/premium à partir de 2,50$ pour augmenter la limite de 100 chansons.",
+    "unknownCommand": "Oh non ! Je ne connais pas cette commande. Si cette erreur persiste, veuillez contacter le support.",
+    "unknownButton": "Oh non ! Je ne connais pas ce bouton. Si cette erreur persiste, veuillez contacter le support.",
+    "unknownSelectMenu": "Oh non ! Je ne connais pas ce menu de sélection. Si cette erreur persiste, veuillez contacter le support.",
+    "unknownModal": "Oh non ! Je ne connais pas ce formulaire. Si cette erreur persiste, veuillez contacter le support.",
+    "unknownError": "Oh non ! Il y a eu un problème. Si cette erreur persiste, veuillez contacter le support.",
+    "serverOnly": "Cette fonctionnalité ne peut être utilisée que dans les serveurs.",
+    "needVoice": "Vous devez être dans un salon vocal pour utiliser cette fonctionnalité.",
+    "cannotJoin": "Je ne peux pas rejoindre ce salon vocal. Veuillez vous assurer que j'ai les permissions nécessaires et réessayez.",
+    "noPlayer": "Il n'y a pas de lecteur audio sur ce serveur. Utilisez {{command:join}} pour en créer un !",
+    "emptyQueue": "La file d'attente est vide. Utilisez {{command:play}} pour ajouter des chansons !",
+    "notSameVoiceChannel": "Vous devez être dans le même salon vocal que moi pour utiliser cette fonctionnalité.",
+    "notInSongRequestChannelUseAble": "Vous ne pouvez pas utiliser cette commande à l'intérieur du salon de demande de chanson.",
+    "botNoPermission": "Je n'ai pas les permissions nécessaires pour effectuer cette action. J'ai besoin de la permission {missingPerms} dans ce salon. Veuillez vous assurer que j'ai les permissions nécessaires et réessayez.",
+    "userNoPermission": "Vous n'avez pas les permissions nécessaires pour effectuer cette action. Vous avez besoin de la permission {missingPerms} dans ce salon.",
+    "djOnly": "Seuls les DJ peuvent utiliser cette fonctionnalité.",
+    "notAllowedToJoinBecauseOfGuildSettings": "Je ne suis pas autorisé à rejoindre ce salon vocal à cause des paramètres du serveur.",
+    "cooldownMessage": "Vous êtes en attente. Veuillez attendre {time} avant d'utiliser à nouveau cette fonctionnalité.",
+    "poweredBy": "Propulsé par Betty.cx",
+    "invite": "Inviter",
+    "support": "Support",
+    "website": "Site web",
+    "subscriptions": "Gérer les abonnements",
+    "premium": "Premium",
+    "listenHere": "Écouter ici",
+    "nowPlaying": "Joue en ce moment **{songName} - {artist}**",
+    "playing": "En train de jouer {songName} - {artist}",
+    "requestedBy": "Demandé par  {requester}",
```

Comment:
2 space wanted here?

Reply:
yeah my bad, i also removed another one, should be good now